image/templates/helm/shared/templates/_scanner-v4_init.tpl.htpl

```diff
@@ -193,13 +153,20 @@
 
 {{/* Provide some human-readable feedback regarding the Scanner V4 configuration to the user installing the Helm chart. */}}
 {{- if and (get $components "indexer") (not (get $components "matcher")) -}}
-  {{- $_ = set $._rox.scannerV4 "_installMode" "indexer-only" -}}
+  {{- $_ = set $._rox.scannerV4 "_installMode" "indexer" -}}
 {{- else if and (not (get $components "indexer")) (get $components "matcher") -}}
-  {{- $_ = set $._rox.scannerV4 "_installMode" "matcher-only" -}}
+  {{/* Just here for completeness, not allowed currently. */}}
+  {{- $_ = set $._rox.scannerV4 "_installMode" "matcher" -}}
 {{- else if and (get $components "indexer") (get $components "matcher") -}}
-  {{- $_ = set $._rox.scannerV4 "_installMode" "indexer-and-matcher" -}}
+  {{- $_ = set $._rox.scannerV4 "_installMode" "indexer and matcher" -}}
 {{- else -}}
-  {{- $_ = set $._rox.scannerV4 "_installMode" "none" -}}
+  {{- $_ = set $._rox.scannerV4 "_installMode" "" -}}
```

Comment:
I think "none" was better. Empty string might be a bit confusing

Reply:
This won't be printed to the user. See the `NOTES` file. If it is the empty string then a hand-crafted message is printed for the user.

Reply:
is empty string "" also a default value? So if "_installMode" is not set it will be also ""?

Reply:
At least I am using it here as a default value. I think helm templating has some notion of "uninitialized variable" -- in the chart we are checking for this using the very simple and straightforward code piece:

```
{{ if kindIs "invalid" someVariable }}
 ...
{{ end }}
```

and I thought checking for `eq "" someVariable` might be much easier to comprehend. If you are asking what would be the idiomatic way to handle this, tbh, I don't know.

Reply:
will resole @kurlov, hope I have addressed your remark sufficiently. If not, please ping me again.